python/undo/State.py

```diff
@@ -18,13 +18,6 @@ def __init__(self, state_type: StateType, variable_values: list[VariableValue] =
         self.state_type = state_type
         self.variable_values = variable_values
 
-    def get_apply_commands(self) -> str:
-        output = ""
-        for variable_value in self.variable_values:
-            output += variable_value.get_apply_command()
-        output += "\n"
-        return output
-
     def has_un_collapsible_difference(self, other: Self) -> bool:
```

Comment:
Should we be removing the collapsible_difference aspects too? Since we wont care about storing duplicated states in the history (since we wont be storing states in the history at all)